tests/e2e/run/all.yml

```diff
@@ -0,0 +1,47 @@
+- name: Include network tasks
```

Comment:
Since the e2e test is meant to be an integrated test rather than a bunch of disjoint ones like func tests, we don't need to separate between resource types here. We can have seed.yml, run.yml, clean.yml without further splitting. It's probably going to be easier to get oriented in the test to have them less "exploded". WDYT?